issues.md

```diff
@@ -1,156 +1,150 @@
+Do the following:
 
-### Discussions
+- Before filing an issue, consider if you need to start a [discussion](https://github.com/mdn/mdn-community/discussions) in the MDN project on GitHub. Use discussions to gain different viewpoints and to converge on an agreed upon course of action. This helps to keep issues focused and productive.
+- Before filing an issue, read our [contribution guide](https://github.com/mdn/content/blob/main/CONTRIBUTING.md) to try to fix the problem yourself.
+- If you have a question, you can ask using mechanisms like [chat rooms](https://chat.mozilla.org/#/room/#mdn:mozilla.org) or [forums](https://discourse.mozilla.org/c/mdn/236) without filing an issue.
 
-- Comments in an issue should be limited to details or context that helps resolve the issue.
-  If a discussion needs to take place to clarify an issue, or if a discussion begins, it should be moved to a GitHub discussion.
-- If an issue has no clear consensus on how to resolve it, it should be moved to a discussion.
-- If the requirements for completing the task expand while it's being resolved, or if the work is unclear, a discussion must be used to flesh out details for clarity.
+Avoid doing the following:
 
-[Issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues) are used to track all bugs and work that has a clear actionable outcome. If you have found a bug with either our content or the platform, please search current open issues against the [relevant repository](/en-US/docs/MDN/Community/Contributing/Our_repositories) to ensure someone has not already reported the issue. If the issue is new, please file an issue using the relevant template available in the repository.
+- Complicating issues by trying to discuss multiple topics or by making off-topic comments.
+- Opening lots of issues asking vague questions.
+- Asking questions without trying to solve the problem yourself first.
 
-> **Note:** If an issue has a triage label, we haven't reviewed it yet, and you shouldn't begin work on it.
+## Guidelines for reporting an issue
 
-If the issue you are filing is not to report a bug, please ensure that it lists actionable tasks or a clear outcome. For example:
+[Issues](https://docs.github.com/en/github/managing-your-work-on-github/about-issues) are used to track bugs. An issue must be a single actionable task or a collection of multiple, related actionable tasks and must have a clear actionable outcome.
 
-```markdown
-## Remove \{{ warning }} macro from documents
+### Before filing an issue
 
-We should no longer be using the `\{{ warning }}` macro in our documentation.
+If you've found a bug with either the content on MDN or the platform, search the current open issues in the [relevant repository](/en-US/docs/MDN/Community/Contributing/Our_repositories) to ensure that someone has not already reported the issue.
```

Comment:
I would suggest it's more like "verify" first:

```suggestion
If you think you've found a bug with the content on MDN or a display problem, search the current open issues in the [relevant repository](/en-US/docs/MDN/Community/Contributing/Our_repositories) and make sure nobody else has reported the issue.
```

Reply:
Also in general I think we can refer to "the platform" as "display issues" to make it easier to understand

Reply:
@bsmth good idea to clarify the meaning of 'platform'. I'm inclined to use '[look and feel](https://en.wikipedia.org/wiki/Look_and_feel)' to cover both display and functionality.